fe/fe-core/src/main/java/com/starrocks/connector/jdbc/MysqlSchemaResolver.java

```diff
@@ -57,6 +60,31 @@ public Collection<String> listSchemas(Connection connection) {
         }
     }
 
+    @Override
+    public boolean checkSupportPartitionInformation(Connection connection) {
+        String catalogSchema = "information_schema";
+        String partitionInfoTable = "partitions";
+        try (ResultSet catalogSet = connection.getMetaData().getCatalogs()) {
+            while (catalogSet.next()) {
+                String schemaName = catalogSet.getString("TABLE_CAT");
+                if (schemaName.equalsIgnoreCase(catalogSchema)) {
+                    try (ResultSet tableSet = connection.getMetaData().getTables(catalogSchema, null, null,
+                            new String[] {"SYSTEM TABLE", "SYSTEM VIEW"})) {
+                        while (tableSet.next()) {
+                            String tableName = tableSet.getString("TABLE_NAME");
+                            if (tableName.equalsIgnoreCase(partitionInfoTable)) {
+                                return true;
+                            }
+                        }
+                    }
+                }
+            }
+        } catch (SQLException e) {
+            throw new StarRocksConnectorException(e.getMessage());
+        }
+        return false;
+    }
+
```

Comment:
i guess this `connection.getMetaData().getTables(db, null, table, new String[] {"SYSTEM TABLE"})` maybe better, but i have not checked if it works

Reply:
This is a VIEW. According to the test results of MySQL 5.7, this table is in SYSTEM VIEW. In the system table of MySQL, the type of this table is also SYSTEM VIEW. However, the interface description in JDBC only includes SYSTEM TABLE, without SYSTEM VIEW. Therefore, I am not sure if this table is in SYSTEM VIEW in all versions of MySQL, but it must be in SYSTEM TABLE or SYSTEM VIEW.